python_scripts/trees_hyperparameters.py

```diff
@@ -115,12 +115,20 @@ def plot_regression(model, X, y, ax=None):
 plot_classification(tree_clf, data_clf[data_clf_columns],
                     data_clf[target_clf_column])
 plt.legend(bbox_to_anchor=(1.05, 1), loc='upper left')
-_ = plt.title(f"Shallow classification tree with max-depth of {max_depth}")
+plt.title(f"Shallow classification tree with max-depth of {max_depth}")
+plt.show()
+accuracy = tree_clf.score(data_clf[data_clf_columns],
```

Comment:
Here I would integrate the scoring in the plot directly.
We should check if it is better to put it as a title or rather in the legend of the plot.

Reply:
I would have the same comment for the other changes in this file